dvc/command/cache.py

```diff
@@ -18,6 +23,86 @@ def run(self):
         return 0
 
 
+class CmdCacheStatus(CmdBase):
+    def run(self):
+        self.permission_check()
+        self.show_dvc_files_status()
+        return 0
+
+    def permission_check(self):
+        cache_dir = self.config["cache"]["dir"]
+        logger.info("Step 1: Permission Check on: {}".format(cache_dir))
+        logger.info(
+            "\tRead: {}".format(
+                "OK" if os.access(cache_dir, os.R_OK) else "Not OK"
+            )
+        )
+        logger.info(
+            "\tWrite: {}".format(
+                "OK" if os.access(cache_dir, os.W_OK) else "Not OK"
+            )
+        )
+        logger.info(
+            "\tExist: {}".format(
+                "OK" if os.access(cache_dir, os.F_OK) else "Not OK"
+            )
+        )
+
+    def show_dvc_files_status(self):
+        logger.info("Step 2: DVC Files and Cache status.")
```

Comment:
What does "DVC Files" refer to specifically? Maybe
```suggestion
        logger.info("Step 2: .dvc/ internal files and cache status")
```

Reply:
Just making it clear it doesn't refer to .dvc files, or dvc.yaml files, or does it?

Reply:
@jorgeorpinel how about `versioned files and cache status`?

Reply:
OK seeing the table headers I think something like this would be accurate:
```suggestion
        logger.info("Step 2: Cache details of all DVC-tracked files:")
```

Reply:
Or maybe even skip this INFO message? It's what th user is expecting when they run `dvc cache status`

@lsc20051426 could you maybe paste here an example output of this new command would look like? To make sure we're on the same page. Like this

```console
$ dvc cache status
Step 1: ...
...
Step 2: ...
(Table)
```

Thanks

Reply:
@jorgeorpinel 
Currently, the output sample:
```
dvc cache status
Step 1: Permission Check on: xxx/.dvc/cache
	Read: OK
	Write: OK
	Exist: OK
Step 2: DVC Files and Cache status.
File/Directory    Name                   Size    CachePath                                         CacheSize
File              data/images/cat1.jpg   31926   .dvc/cache/c6/b4d357699d641823c537514996a44f      31926
File              data/images/cat2.jpeg  31291   .dvc/cache/27/e39df5f434f513fafda870624e063d      31291
File              samples/output1.txt    10      .dvc/cache/3f/a9c057aa667dd5869a249b13574057      10
Dir               data/images            -       .dvc/cache/0f/56c851f043425b39afb8742a2849c3.dir  -
File              data/model.v1.txt      4       .dvc/cache/48/d6d64719872cef5d4d3541ffc4a4a0      4
```
I think `Cache details of all DVC-tracked files:` is a good INFO message.

Reply:
Thanks for the update!